VirtoCommerce.Platform.Core/Assets/AssetEntrySearchCriteria.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.Platform.Core.Assets
+{
+    public class AssetEntrySearchCriteria
```

Comment:
Should be derived from ValueObject to be able to use GetHashCode() method  for  cache key constructing